packages/ngx-flicking/projects/ngx-flicking/src/lib/ngx-flicking.component.ts

```diff
@@ -72,6 +72,9 @@ import NgxElementProvider from './NgxElementProvider';
     class: 'flicking-viewport',
     style: 'display: block;',
   },
+  styleUrls: [
+    "../../node_modules/@egjs/flicking/dist/flicking.css"
```

Comment:
Could it be this way?

`@egjs/flicking/dist/flicking.css`

Reply:
I've checked that absolute path `@egjs/flicking/dist/flicking.css` doesn't work with the intended path when building the package.
https://github.com/angular/angular-cli/issues/20162